configs/lwip/lwipopts.h

```diff
@@ -45,6 +45,10 @@
 #define CONFIG_DYNAMIC_TICKLESS 0
 #endif
 
+#ifndef ENABLE_AMAZON_COMMON
+#define ENABLE_AMAZON_COMMON 1U
```

Comment:
```suggestion
#define ENABLE_AMAZON_COMMON
```
We don't need to define it with any value since we're checking the definition only, like `#if defined(ENABLE_AMAZON_COMMON)`.

Reply:
in our demo_config.h, we define Macros with a value 0U or 1U, if the value is 0U. The macro is not being used in other files. Is it that since its in the same file we don't need to do here ??